schemas/v1/19_actor_mappings.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+func init() {
+	patches.Register(
+		19,
```

Comment:
Due to this [commit](https://github.com/filecoin-project/lily/pull/1173), we should change the version from 19 to 20.